src/interfaces/ISuperstateToken.sol

```diff
@@ -2,12 +2,118 @@
 pragma solidity ^0.8.28;
 
 import {IERC20Upgradeable} from "openzeppelin-contracts-upgradeable/interfaces/IERC20Upgradeable.sol";
-import {IERC7246} from "src/interfaces/IERC7246.sol";
-import {AllowList} from "src/allowlist/AllowList.sol";
-import {ISuperstateTokenV2} from "./ISuperstateTokenV2.sol";
 import {IAllowListV2} from "src/interfaces/allowlist/IAllowListV2.sol";
 
-interface ISuperstateToken is ISuperstateTokenV2 {
+interface ISuperstateToken is IERC20Upgradeable {
+    // V1 remaining
+
+    /// @dev Event emitted when tokens are minted
+    event Mint(address indexed minter, address indexed to, uint256 amount);
+
+    /// @dev Emitted when the accounting pause is triggered by `admin`.
+    event AccountingPaused(address admin);
+
+    /// @dev Emitted when the accounting pause is lifted by `admin`.
+    event AccountingUnpaused(address admin);
+
+    /// @dev Thrown when a request is not sent by the authorized admin
+    error Unauthorized();
+
+    /// @dev Thrown when an address does not have sufficient permissions, as dictated by the AllowList
+    error InsufficientPermissions();
+
+    /// @dev Thrown when the current timestamp has surpassed the expiration time for a signature
+    error SignatureExpired();
+
+    /// @dev Thrown if the signature has an S value that is in the upper half order.
+    error InvalidSignatureS();
```

Comment:
nit: caps on the S is weird?

Reply:
No, its the S argument for `function isValidSignature(address signer, bytes32 digest, uint8 v, bytes32 r, bytes32 s)`. Also this is a copy paste